### DISCUSSION TITLE
'tuple' object has no attribute 'layer'

### DISCUSSION THREAD

**Discussion by FarzanehMoradkhani** [onnx/onnx#3152] [Q&A]
I got this error when I want to convert my Keras code(LSTM) to onnx, do you know what should I do?

onnx_model = onnxmltools.convert_keras(model, target_opset=7)

Error :
 'tuple' object has no attribute 'layer'

**Comment by askhade** [comment]
This is a converter issue. Please open an issue in converter repo https://github.com/onnx/keras-onnx/issues
